recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: 4abff66014b23c578bd656595769cb446664812b2dfecc0dc2e38af4c03ddc12
+  sha256: 3f7038f3cba84f00e9229078f0ea8dca8983fa2079e1d0457d5082d034829ff9
```

Comment:
This matches [here](https://pypi.org/project/optuna-integration/4.3.0/#optuna_integration-4.3.0.tar.gz).